libs/components/progress-indicator/src/lib/modules/progress-indicator/types/progress-indicator-item-status.ts

```diff
@@ -1,21 +0,0 @@
-export enum SkyProgressIndicatorItemStatus {
```

Comment:
This file is exported from the public exports API and documented here: https://developer.blackbaud.com/skyux/components/progress-indicator-waterfall?docs-active-tab=design. Should we remove this file in a separate "breaking change" pull request?

Reply:
No idea what I was thinking. I put the enum back.